README.md

```diff
@@ -235,6 +235,19 @@ import chartmogul
 chartmogul.Transaction.create(config, uuid='inv_745df1d4-819f-48ee-873d-b5204801e021', data={})
 ```
 
+#### [SubscriptionEvents](https://dev.chartmogul.com/docs/subscription_events)
+
+```python
+import chartmogul
+chartmogul.SubscriptionEvent.all(config)
+chartmogul.SubscriptionEvent.create(config, data={})
+chartmogul.SubscriptionEvent.modify(config, id=73966836, data={
+    'currency' : 'EUR',
+    'amount_in_cents' : 100
+})
+chartmogul.SubscriptionEvent.destroy(config, id=73966836)
```

Comment:
Shouldn't this be `destroy_with_params` with id (or `external_id` and `data_source_uuid`) sent as part of the `data` payload?

Reply:
Yes. My bad, I didn't change readme after I added destroy_with_params and modify_with_params